types/json/constants.go

```diff
@@ -212,12 +212,25 @@ var (
 	ErrInvalidJSONPath = terror.ClassJSON.New(mysql.ErrInvalidJSONPath, mysql.MySQLErrName[mysql.ErrInvalidJSONPath])
 	// ErrInvalidJSONData means invalid JSON data.
 	ErrInvalidJSONData = terror.ClassJSON.New(mysql.ErrInvalidJSONData, mysql.MySQLErrName[mysql.ErrInvalidJSONData])
+	// ErrInvalidJSONPathWildcard means invalid JSON path that contain wildcard characters.
+	ErrInvalidJSONPathWildcard = terror.ClassJSON.New(mysql.ErrInvalidJSONPathWildcard, mysql.MySQLErrName[mysql.ErrInvalidJSONPathWildcard])
+	// ErrInvalidJSONContainsPathType means invalid JSON contains path type.
+	ErrInvalidJSONContainsPathType = terror.ClassJSON.New(mysql.ErrInvalidJSONContainsPathType, mysql.MySQLErrName[mysql.ErrInvalidJSONContainsPathType])
 )
 
 func init() {
 	terror.ErrClassToMySQLCodes[terror.ClassJSON] = map[terror.ErrCode]uint16{
-		mysql.ErrInvalidJSONText: mysql.ErrInvalidJSONText,
-		mysql.ErrInvalidJSONPath: mysql.ErrInvalidJSONPath,
-		mysql.ErrInvalidJSONData: mysql.ErrInvalidJSONData,
+		mysql.ErrInvalidJSONText:             mysql.ErrInvalidJSONText,
+		mysql.ErrInvalidJSONPath:             mysql.ErrInvalidJSONPath,
+		mysql.ErrInvalidJSONData:             mysql.ErrInvalidJSONData,
+		mysql.ErrInvalidJSONPathWildcard:     mysql.ErrInvalidJSONPathWildcard,
+		mysql.ErrInvalidJSONContainsPathType: mysql.ErrInvalidJSONContainsPathType,
 	}
 }
+
+// json_contains_path function type choices
+// See: https://dev.mysql.com/doc/refman/5.7/en/json-search-functions.html#function_json-contains-path
+var (
+	ContainsPathAll = "all"
+	ContainsPathOne = "one"
```

Comment:
const is better and comment should be added.

Reply:
Done